testing/ceph/APKBUILD

```diff
@@ -114,59 +117,69 @@ _sysconfdir=/etc
 _udevrulesdir=/etc/udev/rules.d
 _python_sitelib=/usr/lib/python2.7/site-packages
 
-
-_builddir="$srcdir"/$pkgname-$pkgver
-
 build() {
-   cd "$_builddir"
-   mkdir build
-   cd build
-   cmake .. \
-	   -DCMAKE_INSTALL_PREFIX=$_prefix \
-	   -DCMAKE_INSTALL_LIBDIR=$_libdir \
-	   -DCMAKE_INSTALL_LIBEXECDIR=$_libexecdir \
-	   -DCMAKE_INSTALL_LOCALSTATEDIR=$_localstatedir \
-	   -DCMAKE_INSTALL_SYSCONFDIR=$_sysconfdir \
-	   -DCMAKE_INSTALL_DOCDIR=$_docdir/ceph \
-	   -DCMAKE_INSTALL_MANDIR=$_mandir \
-	   -DWITH_REENTRANT_STRSIGNAL=ON \
-	   -DWITH_THREAD_SAFE_RES_QUERY=ON \
-	   -DWITH_MANPAGE=OFF \
-	   -DWITH_PYTHON3=OFF \
-	   -DWITH_TESTS=$WITH_TESTS \
-	   || return 1
-   make
+	export CEPH_BUILD_VIRTUALENV=$builddir
+
+	mkdir -p $builddir/build
+	cd $builddir/build
+
+	if [ -n "$MAKE_CHECK" ] || [ -n "$CEPH_TEST_PKG" ]; then
```

Comment:
Ya this is not right. WITH_TESTS=off is faster, however that's irrelevant if test package is built by default. I'll ditch the "WITH_TEST" build optionality.